src/js/tracks/text-track.js

```diff
@@ -91,14 +91,21 @@ const loadTrack = function(src, track) {
     // NOTE: this is only used for the alt/video.novtt.js build
     if (typeof window.WebVTT !== 'function') {
       if (track.tech_) {
-        const loadHandler = () => parseCues(responseBody, track);
-
-        track.tech_.on('vttjsloaded', loadHandler);
-        track.tech_.on('vttjserror', () => {
+        // to prevent use before define eslint error, we define loadHandler
+        // as a let here
```

Comment:
I think that this was very likely a bug waiting to happen here since we never cleaned up the `vttjsloaded` and `vttjserror` handlers properly.